events/tests/test_notifications.py

```diff
@@ -192,3 +210,74 @@ def test_occurrence_cancelled_notification(
 
     assert len(mail.outbox) == 1
     assert_mails_match_snapshot(snapshot)
+
+
+@pytest.mark.django_db
+def test_occurrence_reminder_notification(
+    snapshot, notification_template_occurrence_reminder_fi, project,
+):
+    actual_now = now()
+
+    # time frozen so that the Enrolments will get created_at in the past
+    with freeze_time(actual_now - timedelta(days=8)):
```

Comment:
To make this test less likely to break if the setting is changed you could either read the setting (from the settings fixture) or use the settings fixture and just force `KUKKUU_REMINDER_DAYS_IN_ADVANCE` to an expected value.

```python
def test_occurrence_reminder_notification(... settings):
    settings.KUKKUU_REMINDER_DAYS_IN_ADVANCE = 7
```

Reply:
Looks like you have done it already. Please ignore this comment. :sweat_smile: